spark/v3.2/spark/src/test/java/org/apache/iceberg/spark/SparkCatalogConfig.java

```diff
@@ -34,7 +34,8 @@
       "type", "hive",
       "default-namespace", "default",
       "parquet-enabled", "true",
-      "cache-enabled", "false" // Spark will delete tables using v1, leaving the cache out of sync
+      "cache-enabled", "false", // Spark will delete tables using v1, leaving the cache out of sync
+      "cache.expiration-interval-ms", "0" // Cache expiration needs to be disabled if cache is
```

Comment:
I don't think this file needs to change, does it?

Reply:
Not if we're defaulting to zero now (aka off).

This was to make the tests predictable. At least at one point it was an error to have a non-zero cache.expiration-interval-ms with `cache-enabled` as false.

Part of the problem is that we continue to send those properties further down into the call, which is why I normalized them. Let me try removing this now since it's currently defaulted to zero.

Reply:
This has been removed. As cache expiry is off when caching is off.